### PR TITLE
Station localisation updated missing translations

### PR DESCRIPTION
**Pull request by markoaamunkajo**
This file is test and done with ChatGPT

### CODE REVIEW

File: station.localization.json [hunk 1]
```diff
@@ -1378,7 +1378,7 @@
       "sv": "Rel",
       "fr": "Rel",
       "de": "Rel",
-	  "pl": ""
+      "pl": ""
```
Comment (andreevdenis) [inline station.localization.json:1381]: I noticed some changes in the begining of the each PL string. Tabs changes to spaces or something like this?

File: station.localization.json [hunk 2]
```diff
@@ -1678,43 +1678,43 @@
...
+      "pl": "Pobieranie historii przez połączenie Bluetooth jest w toku. Proszę czekać."
     },
     {
       "ident_ios": "TagCharts.AbortSync.Alert.message",
       "ident_android": "",
       "ident_webui": "",
       "en": "Sometimes the history download is slow due to the Bluetooth connectivity. Please wait a moment.",
       "fi": "Historian lataaminen voi kestää pitkään Bluetooth-yhteyden laadusta riippuen. Odota hetki.",
-      "ru": "",
+      "ru": "Пример перевода",
```
Comment (andreevdenis) [inline station.localization.json:1701]: "Translation example"

File: station.localization.json [hunk 3]
```diff
@@ -1678,43 +1678,43 @@
...
+      "pl": "Czasami pobieranie historii jest wolne z powodu połączenia Bluetooth. Proszę czekać chwilę."
     },
     {
       "ident_ios": "TagCharts.AbortSync.Button.title",
       "ident_android": "",
       "ident_webui": "",
       "en": "Abort download",
       "fi": "Peruuta lataaminen",
-      "ru": "",
+      "ru": "Пример перевода",
```
Comment (andreevdenis) [inline station.localization.json:1713]: "Translation example"

File: station.localization.json [hunk 4]
```diff
@@ -4451,7 +4451,7 @@
       "sv": "Kontinuerlig bakgrundsskanning",
       "fr": "Balayage en continu",
       "de": "Kontinuierliches Scannen im Hintergrund",
-	  "pl": "Ciągłe skanowanie w tle"
+      "pl": "Ciągłe skanowanie w tle"
```
Comment (andreevdenis) [inline station.localization.json:4454]: Many unnecessary changes in formating

**Review by andreevdenis — CHANGES_REQUESTED** [review]
Let's remove this kind of translation:
"ru": "Пример перевода",

I can go through RU lines later.

These lines should not be changed at all if no translation is provided:
image

**Comment by markoaamunkajo** [comment]
Not implemented at this point, will close this pull request

**Pull request closed without merging**